kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -298,21 +359,36 @@ define([
         }
 
         // EVENTS
-        // function attachEvents() {
-        //     bus.on('reset-to-defaults', function () {
-        //         widgets.forEach(function (widget) {
-        //             widget.bus.emit('reset-to-defaults');
-        //         });
-        //     });
-
-        //     runtime.bus().on('workspace-changed', function () {
-        //         // tell each input widget about this amazing event!
-        //         widgets.forEach(function (widget) {
-        //             widget.bus.emit('workspace-changed');
-        //         });
-        //     });
-        // }
+        function attachEvents() {
```

Comment:
Activated these events - now the messages get passed along to each widget as expected.